### ISSUE TITLE
Update test for PSL(2,7)

### ISSUE THREAD

**Issue by alex-konovalov** [gap-packages/HeLP#3]
In the master branch of GAP, we have now (see e.g. [here](https://travis-ci.org/gap-system/gap-docker-pkg-tests-master/jobs/476162581)):
```
normaliz found
#I  Testing /home/gap/inst/gap-master/pkg/HeLP-3.4/tst/yes_normaliz.tst
########> Diff in /home/gap/inst/gap-master/pkg/HeLP-3.4/tst/yes_normaliz.tst:
39
# Input is:
HeLP_PQ(C1);
# Expected output:
 #I  The Brauer tables for the following primes are not available: [ 2, 3, 7 ]\
. 
 #I  (PQ) can't be solved, using the given data, for the orders: [ 6 ].
 false
# But found:
#I  The Brauer tables for the following primes are not available: [ 2, 7 ].
#I  (PQ) can't be solved, using the given data, for the orders: [ 6 ].
false
########
4ti2 found
#I  Testing /home/gap/inst/gap-master/pkg/HeLP-3.4/tst/yes_4ti2.tst
########> Diff in /home/gap/inst/gap-master/pkg/HeLP-3.4/tst/yes_4ti2.tst:39
# Input is:
HeLP_PQ(C1);
# Expected output:
 #I  The Brauer tables for the following primes are not available: [ 2, 3, 7 ]\
. 
 #I  (PQ) can't be solved, using the given data, for the orders: [ 6 ].
 false
# But found:
#I  The Brauer tables for the following primes are not available: [ 2, 7 ].
#I  (PQ) can't be solved, using the given data, for the orders: [ 6 ].
false
########
#I  Errors detected while testing package help
#I  RunPackageTests("help", "3.4", "tst/testall.g", "auto"): runtime 20636
```
This happens because the following will work in GAP 4.11:
```
gap> C1 := CharacterTable(PSL(2,7));;
gap> C1 mod 3;
BrauerTable( Group([ (3,7,5)(4,8,6), (1,2,6)(3,4,8) ]), 3 )
```
Just updating the test file to new output will cause tests failing in stable-4.10 branch - it may be better to suppress info messages for this example only.

**Comment by margollo** [comment]
Has been solved in release 3.5. To avoid the printing InfoLevel is decreased to 0 before execution of the relevant commands and increased afterwards to 1 again.

**Comment by alex-konovalov** [comment]
Reopened this - can't see changes in the master branch.

**Comment by alex-konovalov** [comment]
Now properly closed in 28fda0373be66df24ab0038105feb69709803dea
